<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1232</width>
    <height>885</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <widget class="QWidget" name="clerk_info" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1141</width>
      <height>91</height>
     </rect>
    </property>
    <widget class="QLabel" name="clerk_id">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>clerk_ID</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="operations_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>261</width>
      <height>721</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="manageCustomers_radio">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Manage Customers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="viewCustomerFinances_radio">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>View Customer 
 Finances</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="viewCusTransactions_radio">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>View Customer 
Transactions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="viewCusRequests_radio">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>View Customer 
Requests</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="operations_stackedWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>951</width>
      <height>741</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="PManage_customers">
     <widget class="QTableWidget" name="MC_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>781</width>
        <height>571</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>customerID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>firstName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>LastName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>email</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>phoneNum</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>TC</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="deleteCustomer_button">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>590</y>
        <width>141</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Customer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addCustomer_button">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>590</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Customer</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="firstName_textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>590</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>firstName</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="lastName_textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>630</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>lastName</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="phoneNum_textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>670</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>phoneNum</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="email_textEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>590</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>email</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="address_textEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>670</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="TC_textEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>630</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>TC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editCustomer_button">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>590</y>
        <width>141</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit Customer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PCustomer_finances">
     <widget class="QTableWidget" name="finances_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>901</width>
        <height>661</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>customerID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>income</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>expense</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>totalBalance</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="PCustomer_transactions">
     <widget class="QTableWidget" name="transactions_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>941</width>
        <height>691</height>
       </rect>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>65</number>
      </attribute>
      <column>
       <property name="text">
        <string>date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>source customer ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>recepient customer ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>source account</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>recepient account</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>total</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>source balance</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>recepient balance</string>
       </property>
      </column>
     </widget>
     <widget class="QComboBox" name="customer_comboBox">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>10</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PCustomer_requests">
     <widget class="QTableWidget" name="customerRequests_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>911</width>
        <height>581</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>customerID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>requestType</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>account</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="accept_button">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>600</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>accept</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reject_button">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>600</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>reject</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1232</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
